docs/bot-optimization.md

```diff
@@ -278,13 +278,13 @@ Please always check if the `DataProvider` is available to avoid failures during
 
 ``` python
 if self.dp:
-    if dp.runmode == 'live':
-        if ('ETH/BTC', ticker_interval) in self.dp.available_pairs:
-            data_eth = self.dp.ohlcv(pair='ETH/BTC',
-                                     ticker_interval=ticker_interval)
+    if self.dp.runmode == 'live':
```

Comment:
@hroff-1902 This one ... `dp.drymode` does not exist - it needs to be `self.dp` ...